### DISCUSSION TITLE
Questions about the new support for file downloads

### DISCUSSION THREAD

**Discussion by kaiyoma** [cypress-io/cypress#15228] [Questions and Help]
As per the resolution of #949 Cypress now supports file downloads. I've briefly looked over the description and the recipes, but I see a couple obstacles to this being useful:

1. It seems that all downloads go into the same folder, which is inconsistent with other artifacts, like screenshots and videos, which are organized into folders based on the test file name. How do we get downloads to automatically go into those same per-test folders?
2. This feature seems to assume that we know the name of the file being downloaded. The UI that I work on generates a new filename on every download (because it includes the timestamp). How do we get the filename of each download?

**Comment by jennifer-shehane** [comment]
I would suggest opening new issues for feature requests for these things - and explain why you need them, what you're trying to test. We had planned on expanding on file download support, but we won't really build more unless it's requested as something people need.

**Reply by kaiyoma** [reply]
Okay, will do. Thanks so much for your quick response!

**Reply by kaiyoma** [reply]
Filed: #15231 and #15232
